src/chains/ethereum_1/ethTransaction.ts

```diff
@@ -99,18 +99,17 @@ export class EthereumTransaction implements Transaction {
     if (!this._actionHelper) {
       throwNewError('Transaction serialization failure. Transaction has no actions.')
     }
-    const { chain, hardfork, nonce } = this._options
+    const { chainSettings } = this._chainState
+    const { chainForkType } = chainSettings
+    const { chainName, hardFork } = chainForkType
```

Comment:
chainForkType is optional in chainSettings - this code would crash if missing
don't restructure these separately if using null operator is cleaner
use ...
const { chainName, hardFork } = chainSettings ?.chainForkType